src/main/java/org/opensearch/security/configuration/DlsFlsFilterLeafReader.java

```diff
@@ -142,6 +142,7 @@ class DlsFlsFilterLeafReader extends SequentialStoredFieldsLeafReader  {
 
                 if (firstChar == '!' || firstChar == '~') {
                     excludesSet.add(incExc.substring(1));
+                    excludesSet.add(incExc.substring(1) + KEYWORD);
```

Comment:
For my knowledge: What would this change accomplish?

Reply:
When documents are parsed fields are mapped to types, for string mappings a keyword field is automatically created.  By additional excluding keyword fields if you wanted to block a field `actors`, we would also block `actors.keyword` to ensure those field values are not included in queries.